src/app/fdctl/config.c

```diff
@@ -559,9 +564,10 @@ fdctl_obj_align( fd_topo_t const *     topo,
   } else if( FD_UNLIKELY( !strcmp( obj->name, "metrics" ) ) ) {
     return FD_METRICS_ALIGN;
   } else {
-    FD_LOG_ERR(( "unknown object `%s`", obj->name ));
-    return 0UL;
+    ulong align = VAL("align");
```

Comment:
This should happen before the other checks... a/k/a at the top of this function.  If the object already specifies a concrete align, then we don't need to calculate it.

Reply:
done